MANIFEST.in

```diff
@@ -22,4 +22,8 @@ prune   extern/qfr/extern/pybind11/docs
 prune   extern/qfr/extern/pybind11/tests
 
 graft   extern/qfr/extern/pybind11_json
-prune   extern/qfr/extern/pybind11_json/test
\ No newline at end of file
+prune   extern/qfr/extern/pybind11_json/test
+
+graft extern/taskflow
+prune extern/taskflow/docs
+prune extern/taskflow/image
```

Comment:
Are there test directories (or, in general, other directories) that can be excluded as well? The source distribution should be as small as possible.